Formula/jello.rb

```diff
@@ -0,0 +1,25 @@
+class Jello < Formula
+  include Language::Python::Virtualenv
+
+  desc "Filter JSON and JSON Lines data with Python syntax"
+  homepage "https://github.com/kellyjonbrazil/jello"
+  url "https://files.pythonhosted.org/packages/9a/c0/2bf7ca82f2dcae5d0dd4373c31917055e71994e06424446ea3a2302fc25d/jello-1.4.1.tar.gz"
+  sha256 "ff75a75a5cd36c9c3b5916896dd7f1011279206eccbe9e9602daa0c94576b2d8"
+  license "MIT"
+
+  depends_on "python@3.9"
+
+  resource "Pygments" do
+    url "https://files.pythonhosted.org/packages/ba/6e/7a7c13c21d8a4a7f82ccbfe257a045890d4dbf18c023f985f565f97393e3/Pygments-2.9.0.tar.gz"
+    sha256 "a18f47b506a429f6f4b9df81bb02beab9ca21d0a5fee38ed15aef65f0545519f"
+  end
+
+  def install
+    virtualenv_install_with_resources
+    man1.install "jello/man/jello.1"
```

Comment:
It seems like the manpage isn't included within the source archive uploaded to PyPI, so the build errors out at this step. We could add a separate resource for the manpage, but that would cause problems when trying to update the formula with `update-python-resources`.

Reply:
No worries - I see my error. I'll add a MANIFEST.in to my project root and rebuild the pypi package and bump to v1.4.2. Thanks!